update history, license, us (#6)
OSGeo/grass-website: content/about/license.md

layout: "overview"
---

> # Copyright and License Information

Reviewer: ```suggestion
## Copyright and License Information
```

> Geographic Resources Analysis Support System (GRASS) is Copyright, 1999-2018 GRASS Development Team, and licensed under terms of the GNU General Public License (GPL). This includes all software, documentation, and associated materials.

Reviewer: ```suggestion
Geographic Resources Analysis Support System (GRASS) is Copyright, 1999-2019 GRASS Development Team, and licensed under the terms of the GNU General Public License (GPL). This includes all software, documentation, and associated materials.
```

> GRASS GIS >= Version 5 is free software, you can redistribute it and/or modify it under the terms of the [GNU General Public License](http://www.gnu.org/licenses/sgpl-3.0.html) as published by the [Free Software Foundation](https://www.fsf.org/), either version 2 of the license, or (at your discretion) any later version.

Reviewer: ```suggestion
GRASS GIS is free and open source software, you can redistribute it and/or modify it under the terms of the [GNU General Public License](http://www.gnu.org/licenses/sgpl-3.0.html) as published by the [Free Software Foundation](https://www.fsf.org/).
```

> This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the [GNU General Public License](http://www.gnu.org/licenses/gpl-3.0.html) for more details or see the [PDF GPL document](https://grass.osgeo.org/uploads/grass/gpl.pdf).

Reviewer: ```suggestion
This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the [GNU General Public License](http://www.gnu.org/licenses/gpl-3.0.html) for more details.
```

If you are not sure about the differences between public domain software, Free Software and proprietary products, have a look at this webpage: [Categories of Free and Non-Free Software](http://www.gnu.org/philosophy/categories.html).

> Questions regarding details of this policy can be directed to this[ email address](grass-web@lists.osgeo.org) for a prompt response.

Reviewer: ```suggestion
Questions regarding details of this policy can be directed to this [email address](grass-web@lists.osgeo.org).
```

See also the [Frequently Asked Questions about the GNU GPL](http://www.gnu.org/licenses/gpl-faq.html).

> # Citing GRASS GID Software

Reviewer: ```suggestion
# Citing GRASS GIS Software
```

> # Citing GRASS GID Software

Reviewer: ```suggestion
## Citing GRASS GID Software
```

Please cite GRASS when using the software in your work. Here are some choices depending on the version used:

> 

Reviewer: ```suggestion
- GRASS Development Team, 2019. Geographic Resources Analysis Support System (GRASS) Software, Version 7.6. Open Source Geospatial 
Foundation. https://grass.osgeo.org
```

- GRASS Development Team, 2018. Geographic Resources Analysis Support System (GRASS) Software, Version 7.4. Open Source Geospatial 
Foundation. https://grass.osgeo.org

> - GRASS Development Team, 2017. Geographic Resources Analysis Support System (GRASS) Software, Version 7.2. Open Source Geospatial Foundation. https://grass.osgeo.org

Reviewer: ```suggestion
```

> - GRASS Development Team, 2014. Geographic Resources Analysis Support System (GRASS) Software, Version 6.4.4. Open Source Geospatial Foundation. http://grass.osgeo.org

Reviewer: ```suggestion
```

> - GRASS Development Team, 2015. Geographic Resources Analysis Support System (GRASS) Software, Version 7.0. Open Source Geospatial Foundation. http://grass.osgeo.org

Reviewer: ```suggestion
```

- GRASS Development Team, 2015. Geographic Resources Analysis Support System (GRASS) Programmer's Manual. Open Source Geospatial Foundation. Electronic document: http://grass.osgeo.org/programming7/

> For more options, see also: [GRASS GIS Citation Repository](https://grasswiki.osgeo.org/wiki/GRASS_Citation_Repository)

Reviewer: ```suggestion
For more options, see also the [GRASS GIS Citation Repository](https://grasswiki.osgeo.org/wiki/GRASS_Citation_Repository)
```
